src/includes/index.ts

```diff
@@ -0,0 +1,26 @@
+import { combine, Store } from 'effector';
+
+export function includes <T extends string>(
+  a: Store<T>,
+  b: Store<T> | T,
+): Store<boolean>
+export function includes <T extends string | number>(
+  a: Store<Array<T>>,
+  b: Store<T> | T,
+): Store<boolean>
+export function includes <T extends string | number>(
+  a: Store<T | Array<T>>,
+  b: Store<T> | T,
+): Store<boolean> {
+  return combine(a, b, (a, b) => {
+    if (Array.isArray(a)) {
+      return a.includes(b);
+    }
+
+    if (typeof a === 'number') {
+      throw new Error('first argument should be an unit of array or string');
+    }
+
+    return a.includes(b as string);
```

Comment:
`a` must be either an array or a string, so I would write it like this:

```
if (typeof a === 'string') {
      return a.includes(b as string);
}

throw new Error('First argument should be a unit of array or string');
```

I'm not a maintainer, I just decided to leave a comment :)